hermit/signer.py

```diff
@@ -504,6 +505,12 @@ def create_signature(self) -> None:
         if was_signed is False:
             raise HermitError("Failed to sign transaction")
 
+        if minimize_signed_psbt:
+            for inp in self.psbt.psbt_ins:
+                inp.prev_tx = None
+                inp.redeem_script = None
+                inp.named_pubs = {}
+
```

Comment:
For maximum reduction, it'd probably make sense to also blank `prev_out`, `witness_script`, and `extra_map`.